README.md

```diff
@@ -138,47 +144,50 @@ To run go-spacemesh you need to specify the parameters shared between all instan
 
-```bash
-./go-spacemesh --tcp-port [a_port] --config [configFileLocation] -d [nodeDataFilesPath]
-```
+    ```bash
+    ./go-spacemesh --listen [a_multiaddr] --config [configFileLocation] -d [nodeDataFilesPath]
+    ```
 
-##### Example
-Assuming tn1.json is a testnet config file saved in the same directory as go-spacemesh, use the following command to join the testnet. The data folder will be created in the same directory as go-spacemesh. The node will use TCP port 7513 and UDP port 7513 for p2p connections:
+    **Example:**
 
-```bash
-./go-spacemesh --tcp-port 7513 --config ./tn1.json -d ./sm_data
-```
+    Assuming `tn1.json` is a testnet config file saved in the same directory as go-spacemesh, use the following command to join the testnet. The data folder will be created in the same directory as go-spacemesh. The node will use TCP port 7513 and UDP port 7513 for p2p connections:
 
-4. Build the [CLI Wallet](https://github.com/spacemeshos/CLIWallet) from source code and run it:
+    ```bash
+    ./go-spacemesh --listen /ip4/0.0.0.0/tcp/7513 --config ./tn1.json -d ./sm_data
+    ```
 
+4. Build the [CLI Wallet](https://github.com/spacemeshos/CLIWallet) from source code and run it:
 5. Use the CLI Wallet commands to setup accounts, start smeshing and execute transactions.
 
-
 ```bash
 ./cli_wallet
 ```
 
 #### Joining a Testnet (with mining)
+
 1. Run go-spacemesh to join a testnet without mining (see above).
 2. Run the CLI Wallet to create a coinbase account. Save your coinbase account public address - you'll need it later.
 3. Stop go-spacemesh and start it with the following params:
 
-```bash
-./go-spacemesh --tcp-port [a_port] --config [configFileLocation] -d [nodeDataFilesPath] --coinbase [coinbase_account] --start-mining --post-datadir [dir_for_post_data]
-```
+    ```bash
+    ./go-spacemesh --listen [a_multiaddr] --config [configFileLocation] -d [nodeDataFilesPath] --coinbase [coinbase_account] --start-mining --post-datadir [dir_for_post_data]
+    ```
 
-##### Example
-```bash
-./go-spacemesh --tcp-port 7513 --config ./tn1.json -d ./sm_data --coinbase 0x36168c60e06abbb4f5df6d1dd6a1b15655d71e75 --start-mining --post-datadir ./post_data
-```
+    **Example:**
+
+    ```bash
+    ./go-spacemesh --listen /ip4/0.0.0.0/tcp/7513 --config ./tn1.json -d ./sm_data --coinbase 0x36168c60e06abbb4f5df6d1dd6a1b15655d71e75 --start-mining --post-datadir ./post_data
+    ```
 
 4. Use the CLI wallet to check your coinbase account balance and to transact
 
 #### Joining Spacemesh ([TweedleDee](https://testnet.spacemesh.io/#/?id=what-is-spacemesh-01-tweedledee)) Testnet
+
 Find the latest Testnet release in the [releases](https://github.com/spacemeshos/go-spacemesh/releases/) and download the precompiled binary for your platform of choice (or you can compile `go-spacemesh` yourself, from source, using the release tag). The release notes contain a link to a `config.json` file that you'll need to join the testnet.
```

Comment:
@lrettig what should we do about this section?
Should we remove mention of TweedleDee?

+ I haven't seen release notes with a link to the config.json recently. @andres-spacemesh 

Note: approving this PR because this other stuff can be changed later

Reply:
Remove mentions of tweedledee. Maybe remove all mentions of the testnet completely for now. We'll re-add it later when we have new testnet plans in place.